### ISSUE TITLE
Kindle Update

### ISSUE THREAD

**Issue by alanbsmithee** [apprenticeharper/DeDRM_tools#937]
With Catalina I can no longer open old version of Kindle. Will the Calibre work with the latest Kindle? Will it convert mobi or azw to epub and remove DRM?

**Comment by ElleKayEm** [comment]
Post at MobileRead about getting files you can DeDRM from the latest Kindle for Mac:

https://www.mobileread.com/forums/showpost.php?p=3819708&postcount=508

**Comment by alanbsmithee** [comment]
What if I already updated to Catalina before updating Kindle? In the post it stated that one should update to Kindle 1.23.1 then 1.26.1 before updating to Catalina. But I updated the OS without knowing that I could no longer open the old version of Kindle

**Comment by ElleKayEm** [comment]
If you had things working with Kindle for Mac 1.23 or earlier and DeDRM before updating to Catalina, you might still be okay.  But if not, then you'll probably need to access your books another way.  E-ink Kindle, another computer, virtual machine.

**Comment by alanbsmithee** [comment]
After installing the latest calibre plugin and opening a book on Kindle 1.26.1, the designation changed from .kcr back to .azw. I was able to load it into Calibre but the DRM was not removed.

**Comment by mpaezt** [comment]
I wasn't able to update Kindle and Calibre before updating Catalina so, I had no other chance that to convert it in a VM installed on mac. I tried several ways to try to make the DeDRM work on Catalina, nothing happens.

**Comment by ZolaLa9** [comment]
If you had Kindle for Mac 1.23 (and possibly earlier) already working with DeDRM before updating to Catalina, you can try downloading Kindle for Mac 1.26.1 from Amazon at [https://s3.amazonaws.com/kindleformac/55093/KindleForMac-55093.dmg](https://s3.amazonaws.com/kindleformac/55093/KindleForMac-55093.dmg) then copy 1.26.1 over 1.23. 

This has worked for some people and seems to depend on if the leftover files in Library > Application Support > Kindle (or Library > Containers) and also Library > Preferences > calibre > plugins > dedrm.json are enough to make the jump successfully to Catalina. 

If that doesn't work and you have a backup of your Library folders when it was working prior to updating to Catalina, you could try deleting 1.26.1, copying the Application Support > Kindle folder and the dedrm.json file into the corresponding folders in your Catalina machine, then installing 1.26.1, and see if that works to simulate upgrading from 1.23.1 to 1.26.1. 

If you're comfortable with VM, you could create a VM with High Sierra or Mojave, install 1.23.1, get it working with calibre, then copy the Kindle and calibre dedrm.json file to Catalina and see if it will work with 1.26.1.

If you try any of this, I would appreciate hearing what worked and what didn't.

**Comment by alanbsmithee** [comment]
I downloaded 1.23.1 but was not able to install with Catalina. I could only install 1.26.1 and downloaded by using right click>download. I got an azw file to which I was able to add to Calibre but it did not remove the DRM.

**Comment by ElleKayEm** [comment]
1.23 is a 32-bit program so it cannot run on Catalina.  If you had DeDRM working with 1.23 or an earlier version of Kindle for Mac BEFORE Catalina, it might have left a key behind that would work with 1.26.  Were you able to DeDRM Kindle for Mac books on this machine before the update to Catalina?

**Comment by ZolaLa9** [comment]
As ElleKayEm notes, 1.23 will not run on Catalina. Did you have a working set up of Kindle for Mac and Calibre before you updated to Catalina? If so, try installing 1.26.1. The worst that will happen is it will continue not to work.

**Comment by alanbsmithee** [comment]
No, I was happily using 1.7.1. It was working fine as far as DRM removal and conversion so I didn't feel the need to update to 1.23.1.

**Comment by ElleKayEm** [comment]
I might have expected that version to leave behind a key too, but maybe that's too much of a jump.

**Comment by ZolaLa9** [comment]
@alanbsmiththee: Have you even tried installing 1.26.1? 

If you had a working version of Kindle for Mac (1.17?) and Calibre, you can try everything I wrote above with whatever version you had. The worst that will happen is it will continue to not work — which is exactly where you are now.

**Comment by alanbsmithee** [comment]
I'll try it and keep you updated
